### PR TITLE
Add episode #24: Geeksblabla MSS with Yassine El Kachchani

### PR DESCRIPTION
**Pull request by azihassan**
Potential issues :

* I translated "form juridique" into "legal form", but I'm not sure if it's the correct translation
* I probably missed something between the 1:26 and 1:35 marks
* I most likely missed some links

### CODE REVIEW

File: blablas/ep24/index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+podcastUrl: "https://www.facebook.com/sfoukahi/videos/2770524023008256/"
+---
+
+In this episode of GeeksBlabla, we discuss startuping with Yassine, the CEO of United Remote.
+
+## Guests
+
+[Yassine El Kachchani](https://www.facebook.com/kachchani)
+
+[Mohammed Aboullaite](https://www.facebook.com/aboullaite)
```
Comment (yjose) [inline blablas/ep24/index.md]: Delete Mohammed from Guests and add him to the ` Prepared and Presented by `section.

File: blablas/ep24/index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## Notes
+
+If you want contribute in Geksblabla project by adding the notes check [this issue](https://github.com/DevC-Casa/geeksblabla.com/issues/23/)
```
Comment (yjose) [inline blablas/ep24/index.md]: remove this line

File: blablas/ep24/index.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+0:03 - Who is Yassine El Kachchani ?
```
Comment (yjose) [inline blablas/ep24/index.md]: Make sure to update times to correct format `hh:mm` ex: `00:03`

File: blablas/ep24/index.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+0:10 - What lessons did you learn from your very first company ?
+
+0:12 - Brief talk about Lacarteplz.ma
+
+0:16 - How did you come up with the idea of Hidden Founders after two failed companies ?
+
+0:22 - Can you talk a bit about the success of Hidden Founders ?
+
+0:30 - Why did you kill Hidden Founders in favour of United Remote despite the former's success ?
```
Comment (yjose) [inline blablas/ep24/index.md]: favor*

File: blablas/ep24/index.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+0:34 - Does 3% equity translate to 3% of profit or to 3% of shares ?
+
+0:35 - What do you mean by a tiny project ? An MVP, a landing page or what exactly ?
+
+0:37 - Are there Moroccan incubators which offer training or financing that you can recommend ?
```
Comment (yjose) [inline blablas/ep24/index.md]: that offer*

File: blablas/ep24/index.md (hunk 2 above)
Comment (azihassan) [reply blablas/ep24/index.md]: The "If you want to contribute" line ?

Comment (yjose) [reply blablas/ep24/index.md]: Yess

File: blablas/ep24/index.md (hunk 3 above)
Comment (azihassan) [reply blablas/ep24/index.md]: [This issue](https://github.com/DevC-Casa/geeksblabla.com/issues/23) mentions "h:min" in the #Notes section though, is that cool ?

Comment (yjose) [reply blablas/ep24/index.md]: Hmmm, ok.

File: blablas/ep24/index.md (hunk 5 above)
Comment (azihassan) [reply blablas/ep24/index.md]: [Done](https://github.com/DevC-Casa/geeksblabla.com/commit/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fR40) :heavy_check_mark:

File: blablas/ep24/index.md (hunk 4 above)
Comment (azihassan) [reply blablas/ep24/index.md]: [Done](https://github.com/DevC-Casa/geeksblabla.com/commit/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fR34) :heavy_check_mark:

File: blablas/ep24/index.md (hunk 2 above)
Comment (azihassan) [reply blablas/ep24/index.md]: [Done](https://github.com/DevC-Casa/geeksblabla.com/commit/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fL24-L25) :heavy_check_mark:

File: blablas/ep24/index.md (hunk 1 above)
Comment (azihassan) [reply blablas/ep24/index.md]: Removed from here: https://github.com/DevC-Casa/geeksblabla.com/pull/73/commits/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fL20-L21

Added here: https://github.com/DevC-Casa/geeksblabla.com/pull/73/commits/e391833e02fee3ee8151c297b56db24b4c62b648#diff-061138f907d11d2472101c5ef1450c5fR86-R87

**Comment by yjose** [comment]
@all-contributors  please add  @azihassan  for content

**Pull request merged**